packages/ckeditor5-dev-env/lib/release-tools/utils/getnewreleasetype.js

```diff
@@ -5,7 +5,8 @@
 
 'use strict';
 
-const { commitTypes, transform } = require( '../changelog/writer-options' );
+const { availableTypes } = require( './transformcommit' );
```

Comment:
The name `availableTypes` becomes too generic when it's used without the `transformCommit` context.